media/css/pocket/components/error-page.scss

```diff
@@ -0,0 +1,10 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+@import '../utils/variables';
+@import '../includes/platform-nav';
+@import '../includes/platform-footer';
+
```

Comment:
**Suggestion (non-blocking):** Typically in Pocket and the rest of our redesign, most of our inline anchor tags are either:
- colored grey with teal in pseudo states (i.e. https://dev.tekcopteg.com/en/contact-info/) OR
- colored teal with grey in pseudo states (i.e. https://dev.tekcopteg.com/en/about/)

I think it would be nice and consistent to stick to the color scheme. I see in the [original 404 page](https://getpocket.com/en/404) it's teal (with no color change in pseudo state). How about we make the anchor tags colored teal with the grey hover states?

Reply:
Good catch! Yes will line up with with about styles